### ISSUE TITLE
image elements do not have [alt] attributes

### ISSUE THREAD

**Issue by jackyzha0** [getreflect/get-reflect#30]
Informative elements should aim for short, descriptive alternate text. Decorative elements can be ignored with an empty alt attribute.

Failing elements
* img.transition
* img.download-icon
